src/test/resources/com/sap/oss/phosphor/fosstars/model/rating/oss/OssArtifactSecurityRatingTestVectors.yml

```diff
@@ -310,6 +320,9 @@ elements:
   - type: "StandardTestVector"
     values:
       - type: "VulnerabilitiesValue"
+        feature:
```

Comment:
This and some other test cases have a non-empty `VULNERABILITIES_IN_ARTIFACT` but `VULNERABILITIES_IN_PROJECT` is empty (it comes from the `defaults`). This is a bit unrealistic because `VULNERABILITIES_IN_PROJECT` should contain all `VULNERABILITIES_IN_ARTIFACT`.